configuration/sitemaps.md

```diff
@@ -506,29 +529,33 @@ visibility=[Day_Time=="Morning", Day_Time=="Afternoon", Temperature>19]
 ```
 
 If any one of the comparisons is evaluated as `true`, the Item will be visible, otherwise it will be hidden.
-It is important to note, that it is not possible to decide visibility on more than one condition at the same time.
-The third example might be the visibility of a sprinkler control.
-The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
-To achieve more complex conditions, you will benefit from defining a helper Item and a rule to set it.
+It is important to note that visibility may only be determined based on one condition at a time.
 
-Valid comparison operators are:
+The third example might control the visibility of a lawn sprinkler control.
+The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C.
```

Comment:
... a combination of multiple conditions is not supported...

Reply:
Dropping the "Third example".

Reply:
Not sure if we are on the same page. The example is important, I just think that it would be clearer if we specifically stated, that OR means that combinations are indeed not supported

Reply:
Yes, we are not on the same page.

So I dropped the example that is referred to by: "The control will be visible if it is Morning *OR* if it is Afternoon *OR* if the temperature is above 19°C."

I thought you were saying that this example with the OR logic is broken and therefore should be removed.  In your note above, I am failing at parsing your sentence, "it would be clearer if we specifically stated, that OR means that combinations are indeed not supported".  What do you mean by this?

Reply:
IMPORTANT - is this sentence, which is currently submitted as a change, correct?

"It is important to note that visibility may only be determined based on one condition at a time."

FWIW, if it is correct, I would reword this to, "It is important to note that visibility is determined based upon only one condition at a time."